crazyflie_sim/crazyflie_sim/visualization/rviz.py

```diff
@@ -1,32 +1,36 @@
+from __future__ import annotations
+
 from geometry_msgs.msg import TransformStamped
 from tf2_ros import TransformBroadcaster
 from rclpy.node import Node
+from ..simtypes import State, Action
 
 
 class Visualization:
```

Comment:
Same here :) Comment about what the class is doing

Technically this is more like a transform broadcaster and it could be read out by anything that can read that, but RVIZ is used by standard for that so I wouldn't know any other reason why to call it differently